### ISSUE TITLE
`del` does not work properly with nested objects

### ISSUE THREAD

**Issue by fidian** [alexxander/jq-tools#1]
Input: `{"x":{"y":"keep","z":"remove"}}`
Query: `.x | del(.z)`

I've tested this with the published version 0.0.9 using the below command.

```
$ echo '{"x":{"y":"keep","z":"remove"}}' | npx @jq-tools/jq@0.0.9 '.x | del(.y)'
{
  "y": "keep",
  "z": "remove"
}
```

**Comment by alexxander** [comment]
I looked into this a little bit. The `del` filter is actually implemented as:
```shell
def del(f): delpaths([path(f)]);
```

And it appears that the problem is in the implementation of the native `path/1` filter.

The following should resolve to `["y"]`, but it resolves to `["x", "y"]`.

```shell
echo {} | npx @jq-tools/jq '{} |.x| path(.y)'
```

**Comment by alexxander** [comment]
Fixed in v0.0.10

**Comment by fidian** [comment]
Thank you very much! Your response time is amazing.
